recipes/at-spi2-core/all/conanfile.py

```diff
@@ -11,6 +15,7 @@ class AtSpi2CoreConan(ConanFile):
     homepage = "https://gitlab.gnome.org/GNOME/at-spi2-core/"
     license = "LGPL-2.1-or-later"
     generators = "pkg_config"
+    deprecated = True
```

Comment:
As seen in the linked PR https://github.com/conan-io/conan/pull/13991, the deprecated field can contain custom text to show to the user, which is a great opportunity to show them _why_ they should use the newer version. So something like your comment would be great!

```suggestion
    deprecated = "Consumers should migrate to at-spi2-core/[>=2.45.1], which includes atk and at-spi2-atk"
```